src/org/usfirst/frc/team2706/robot/commands/teleop/ArcadeDriveWithJoystick.java

```diff
@@ -2,25 +2,29 @@
 
 import org.usfirst.frc.team2706.robot.Robot;
 
-import edu.wpi.first.wpilibj.GenericHID;
+import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.command.Command;
 
 /**
  * Have the robot drive arcade style using the Xbox Joystick until interrupted.
  */
 public class ArcadeDriveWithJoystick extends Command {
     
-	public final GenericHID joystick;
+	protected Joystick joystick;
 	
 	public ArcadeDriveWithJoystick() {
 		this(Robot.oi.getDriverJoystick());
 	}
 	
-    public ArcadeDriveWithJoystick(GenericHID joystick) {
```

Comment:
Wait sorry, Joystick is supposed to be an actual Joystick that does stuff, while GenericHID is just a abstract class that Joystick extends